setup.py

```diff
@@ -0,0 +1,57 @@
+import pathlib
+
+from skbuild import setup
+
+def exclude_static_libraries(cmake_manifest):
+    return list(filter(lambda name: not (name.endswith(".a") and not name.endswith(".lib")), cmake_manifest))
```

Comment:
`not (name.endswith(".a") and not name.endswith(".lib"))` seems redundant at best and a bug/typo at worst, the same name won't end with `.lib` if is already ends with `.a`

Is it meant to check that it doesn't end in neither `.a` nor `.lib`?
```suggestion
    return list(filter(lambda name: not name.endswith(".a") and not name.endswith(".lib"), cmake_manifest))
```
or same with list comprehension
```suggestion
    return [name for name in cmake_manifest if not name.endswith(".a") and not name.endswith(".lib")]
```

Reply:
I think the idea is to exclude static libs from the final wheel. Not sure about the code, it's coming from the initial PR from @sitic

Reply:
Actually depending on what the python module should contain (only lib bindings? plugins? app? configs? ...) there may be more to filter out, see other comment